### PR TITLE
Editing QTS page content

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/8PPGnp7p/4046-improve-qts-page

### Context

Our current QTS page content is quite sparse and in places, inaccurate. We should improve the content by adding more information around how to get QTS.

### CODE REVIEW

File: app/views/content/what-is-qts.md [hunk 1]
```diff
@@ -26,26 +31,43 @@ keywords:
...
+Some schools can employ teachers without QTS (for example, [private schools](https://www.gov.uk/types-of-school/private-schools) and [academy schools](https://www.gov.uk/types-of-school/academies)) but many still use it to assess the quality of candidates.
 
-QTS is a teaching standard that is respected across the world. Having QTS means you can move around the country and teach in any state school.
+You do not need QTS to teach in [further education](/become-a-further-education-teacher) or work in [early years education](/early-years-teaching-training).
 
-## How to get QTS
+$international-content$
 
-All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+## Get QTS through teacher training
```
Comment (gemmadallmandfe) [inline app/views/content/what-is-qts.md:42]: @EllieNodder Is this section slightly confusing with three different links? Particularly the first and third link, which sound quite similar:

* Find out more about teacher training
* postgraduate certificate in education (PGCE)
* Find out how to get QTS through teacher training

I know that the destination pages have different content but I found it a bit confusing - I'm not sure as a user I would know quite which one to pick

File: app/views/content/what-is-qts.md [hunk 2]
```diff
@@ -26,26 +31,43 @@ keywords:
...
+* for a significant amount of time (for example, 2 years)
+* in more than one school
+
+[Find out how to get QTS through assessment only](/train-to-be-a-teacher/if-you-have-experience).
+
+## Get QTS if you qualified as a teacher outside England
+
+If you trained to teach in Wales, your QTS will be automatically recognised in England and awarded by the [Education Workforce Council (EWC)](https://www.ewc.wales/site/index.php/en/).
+
+If you trained to teach in Scotland, Northern Ireland or outside the UK, you may be able to apply for QTS in England without any further fees or training.
```
Comment (gemmadallmandfe) [inline app/views/content/what-is-qts.md:71]: @EllieNodder Is the 'outside the UK' obvious enough? I read Scotland and Northern Ireland only when first scanning through, and then wondered why we didn't talk about non-UK people :-)

Might just be me though!

File: app/views/content/what-is-qts.md (hunk 1 above)
Comment (EllieNodder) [reply app/views/content/what-is-qts.md:42]: @gemmadallmandfe good point. How about we ditch the first teacher training link and put that in the related content section instead, replacing the blog (bottom link)?

The main CTA for that section should be the final how to get QTS through teacher training link (and there's a link on that page to the other one anyway)

File: app/views/content/what-is-qts.md (hunk 2 above)
Comment (EllieNodder) [reply app/views/content/what-is-qts.md:71]: @gemmadallmandfe 
```suggestion
If you trained to teach anywhere else outside England (including countries outside the UK), you may be able to apply for QTS without any further fees or training.
```
do you think this works better?

Comment (gemmadallmandfe) [reply app/views/content/what-is-qts.md:71]: How about 'If you trained to teach in Scotland, Northern Ireland or in a country outside the UK,....'

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Comment by EllieNodder** [comment]
Reviewed by Gaby (policy)

**Pull request merged**